spectacles/cli.py

```diff
@@ -577,6 +579,22 @@ def _build_content_subparser(
         help="Exclude errors found in content in personal folders.",
     )
 
+    subparser.add_argument(
+        "--exclude-folders",
+        type=int,
+        nargs="+",
+        help="Exclude errors found in folders specified by id.",
+        default=[],
+    )
+
+    subparser.add_argument(
+        "--folders",
+        type=int,
+        nargs="+",
+        help="Include errors found in folders specified by id, takes precedence over --exclue-personal or --exclude-folders.",
```

Comment:
Typo in this line, also exclude should take precedence as we discussed @DylanBaker.